specification/monitor/resource-manager/readme.python.md

```diff
@@ -20,6 +20,7 @@ Generate all API versions currently shipped for this package
 
 ```yaml $(python) && $(multiapi)
 batch:
+  - tag: package-2019-10-17-preview-only
```

Comment:
need this one here: package-2019-10-preview?

Reply:
why? I just change the tag name to follow the previous practice

Reply:
you changed tag name from Tag: 2019-10-17-preview-only to Tag: package-2019-10-preview in readme.md
so, in readme.python.md, tag here shoud use package-2019-10-preview defined in readme.md
pls duble confirm

Reply:
Yes, you are correct. Just fix it